apisix/plugins/limit-conn/init.lua

```diff
@@ -119,7 +119,11 @@ function _M.decrease(conf, ctx)
             if ctx.proxy_passed then
                 latency = ctx.var.upstream_response_time
             else
-                latency = ctx.var.request_time - delay
+                if ctx.var.request_time ~= nil then
+                    latency = ctx.var.request_time - delay
+                else
+                    latency = ctx.var.upstream_response_time
```

Comment:
The stream module doesn't have the `upstream_response_time` variable too

Reply:
Are you sure, I tested it locally and it worked 🤔

Reply:
These two variables can be nil, but the calculation will throw an exception. In fact, the limit-conn plug-in is invalid in stream mode. Can we return directly here?